src/pseudopeople/configuration/generator.py

```diff
@@ -66,6 +66,17 @@
             },
         },
     },
+    # No noise for date in tax w2 1099 dataset
+    DATASETS.tax_w2_1099.name: {
```

Comment:
Did you mean W2 or 1040 here (or both)?

Reply:
W2. There is no date of birth in the 1040: https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#id120